conslogging/conslogging.go

```diff
@@ -74,12 +75,14 @@ func (cl ConsoleLogger) clone() ConsoleLogger {
 		nextColorIndex: cl.nextColorIndex,
 		prefixPadding:  cl.prefixPadding,
 		mu:             cl.mu,
+		bb:             cl.bb,
 	}
 }
 
 // WithPrefix returns a ConsoleLogger with a prefix added.
 func (cl ConsoleLogger) WithPrefix(prefix string) ConsoleLogger {
 	ret := cl.clone()
+	ret.errW = io.MultiWriter(cl.errW, cl.bb.PrefixWriter(prefix))
```

Comment:
Ran some experiments. We don't appear to ever call this a second time, but past performance does not predict future results (or at least thats what Vanguard tells me). To protect against this, we would need to make a new multiwriter using the original intended `os.Stderr` and the new prefix.

Reply:
We should probably allow this to be called multiple times cleanly, for that "future" possibility :-)